### PR TITLE
fix: patched ServiceKafkaQuotaDescribe parameters

### PR DESCRIPTION
**Pull request by vmyroslav**
## About this change - What it does
- patched ServiceKafkaQuotaDescribe parameters. 
- limited the "limit" query param that added to the queries.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -185,18 +188,36 @@ func isEmpty(a any) bool {
...
-func fmtQuery(query ...[2]string) string {
+func fmtQuery(operationID, method string, query ...[2]string) string {
 	q := make(url.Values)
+
+	// Add all provided query parameters
 	for _, v := range query {
-		q.Add(v[0], v[1])
+		if v[0] != "" { // Skip empty keys
+			q.Add(v[0], v[1])
+		}
```
Comment (byashimov) [inline client.go:198]: Empty key is a valid value for query params.

File: client.go [hunk 2]
```diff
@@ -185,18 +188,36 @@ func isEmpty(a any) bool {
...
+func shouldAddDefaultLimit(operationID, method string, q url.Values) bool {
+	var operationsWithoutLimit = []string{
+		"ServiceKafkaQuotaDescribe",
+		"ServiceKafkaQuotaDelete",
+	}
+
+	return !q.Has("limit") &&
+		method == http.MethodGet &&
+		!slices.Contains(operationsWithoutLimit, operationID)
+}
```
Comment (byashimov) [inline client.go:223]: Some methods might use POST to list items. So I would got with operation id filtering only.

nit: I believe switch would do the job too. We should eventually remove this hack and explicitly add the "limit" parameters where they belong.

**Review by byashimov — CHANGES_REQUESTED** [review]

File: client.go (hunk 1 above)
Comment (vmyroslav) [reply client.go]: upd

File: client.go (hunk 2 above)
Comment (vmyroslav) [reply client.go:220]: upd

**Review by byashimov — APPROVED** [review]

**Pull request merged**